src/main/java/io/odpf/firehose/sink/objectstorage/ObjectStorageSink.java

```diff
@@ -18,23 +22,39 @@
 
 public class ObjectStorageSink extends AbstractSink {
 
+    private final boolean isFailOnDeserializationError;
     private final WriterOrchestrator writerOrchestrator;
-    private final OffsetManager offsetManager = new OffsetManager();
```

Comment:
We can keep this. and remove from the constructor. so there will be no change in the constructor.

Reply:
moved to constructor

Reply:
keep the line same, it's adding additional change.